server/portal/apps/workspace/api/views.py

```diff
@@ -12,15 +12,14 @@
 from django.contrib.auth.decorators import login_required
 from django.utils.decorators import method_decorator
 from django.urls import reverse
-from portal.utils.translations import get_jupyter_url
+# from portal.utils.translations import get_jupyter_url
```

Comment:
We can remove `get_jupyter_url` and associated code entirely, it doesn't appear to be used on the frontend